tests/test_web_sendfile_functional.py

```diff
@@ -165,7 +165,9 @@ def handler(request):
     resp.close()
 
     resp = yield from client.get('/', headers={'If-Modified-Since': lastmod})
+    body = yield from resp.read()
     assert 304 == resp.status
+    assert b'' == body
```

Comment:
Hello @es1024 ,

According to what I read on the RFC:
 *  http://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.3
 *  http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.3.5

The Content-Length header should not even be there. Can you add an assert to enforce it?